.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,169 @@
+#
+# Copyright (c) 2021-2023 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Empty Workspace test suite on minikube
+
+# Trigger the workflow on pull request
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+      - 7.**.x
+    paths:
+      - 'tests/e2e/**'
+      - '.github/workflows/pr-check.yml'
+env:
+  LOCAL_TEST_DIR: /tmp
+
+jobs:
+  pr-check:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+
+      - name: Branch name
+        run: |
+          echo running on PR ${GITHUB_REF}, branch - ${GITHUB_HEAD_REF}
+          echo "pr_number=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')" >> "$GITHUB_ENV"
+
+      - name: Configuring nodejs 16.x version
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Pull Che-Code Docker image
+        run: |
+          #
+          # pull che-code:latest docker image
+          #
+          docker pull quay.io/che-incubator/che-code:latest
+
+      - name: Start minikube
+        id: run-minikube
+        uses: che-incubator/setup-minikube-action@next
+        with:
+          minikube-version: v1.23.2
+
+      - name: Install chectl
+        run: bash <(curl -sL https://www.eclipse.org/che/chectl/) --channel=next
+
+      - name: Deploy Che
+        run: |
+          #
+          # load Che-Code image into minikube
```

Comment:
It has to be next version for Eclipse Che Next (--channel=next in line 57 above).

Therefore, correct image is address: quay.io/che-incubator/che-code:next .

Reply:
we tested that test commands still works correctly, not our product. I think we can use stable version here.

Reply:
It's about checking PRs into the `main` branch, isn't it?
And `main` branch is dedicated to testing Eclipse Che Next.

So, it's reasonable to run PR check against Eclipse Che Next.